README.md

```diff
@@ -9,3 +9,85 @@ Based on https://github.com/exasol/error-reporting-java
 
 Major difference is this library is simplified by the string interpolation feature available in the C# language.
 
+## Installation
+
+- Make sure you have the .NET core SDK ( use the dotnet SDK installation step on GitHub runners ).
+
+- You'll need to add the Exasol Github NuGet repository to the NuGet package manager on your local machine or the GitHub CI runner, like so: 
+   - On GitHub CI runners:
+	`dotnet nuget add source --username <username/or ci user> --password ${{ secrets.GITHUB_TOKEN }} --store-password-in-clear-text --name github "https://nuget.pkg.github.com/EXASOL/index.json"`
+   - You can use a GitHub PAT on your local system:
+    `dotnet nuget add source --username <username/or ci user> --password <yourPAT> --store-password-in-clear-text --name github "https://nuget.pkg.github.com/EXASOL/index.json"`
+
+- You can then add the package to your projects: `$ dotnet add <PROJECT> package error-reporting-csharp --version 0.2.0`
+## Usage
+
+### Simple Messages
+
+Simple error messages consist out of the message builder helper method which specifies an identifier and one or more messages describing the error.
+```csharp
+ExaError.MessageBuilder("E-TEST-1").Message("Something went wrong.").ToString();
+```
+
+Result: `E-TEST-1: Something went wrong.`
+
+### Parameters/String interpolation
+
+Since C# has string interpolation you should use that in messages and mitigations where needed, instead of parameters (which are currently not supported):
+
+```csharp
+ExaError.MessageBuilder("E-TEST-2")
+    .Message($"Unknown input: '{input}'.").ToString();
+```
+
+Result:
+
+    E-TEST-2: Unknown input: 'unknown'.`
```

Comment:
To keep the consistency, could you please also put this line into single quotes (as the above one)?

Reply:
Maybe putting the first one into block quote should be fine also.